packages/payment-integration-api/src/payment/payment.ts

```diff
@@ -308,3 +309,15 @@ export interface StripeV3FormattedPayload {
         token: string;
     };
 }
+
+export interface TDOnlineMartFormattedPayload {
+    cart_id?: string;
```

Comment:
I would recommend to create some main type like TokenizedCardFormattedPayload
That would be use by at least Stripe and TDbank.
Please look how FormattedVaultedInstrument works https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/payment-integration-api/src/payment/payment.ts#L221

Reply:
I added the changes. We wasn't sure what BE will exactly need, thats why I added more fields.
For now I left only:
```
credit_card_token?: {
        token?: string;
    };
```
I will add more additional fields when we will work on certain part of the integration.

Reply:
`BlueSnapDirectCreditCardInstrument` has the same shape as you have, so it might be better to update it as well.
<img width="440" alt="Screenshot 2023-12-08 at 13 04 23" src="https://github.com/bigcommerce/checkout-sdk-js/assets/25133454/b356038f-5478-4c9e-9c22-edea5d8dee72">

Reply:
Also I might recommend to check https://github.com/bigcommerce/bigpay-client-js repo (it maps the payload that we send from checkout sdk to bigpay payload).

So as we can see on this line: https://github.com/bigcommerce/bigpay-client-js/blob/a0c61393a467b6e866fd1c5d60bc2f833f6be87d/src/payment/v1/payment-mappers/payment-mapper.js#L66, you can pass `nonce` to the payment payload and it will automatically convert it to credit_card_token object.

Therefore, you can use NonceInstrument instead, hence - no need in this PR at all